generators/dart/math.js

```diff
@@ -275,81 +275,81 @@ Blockly.Dart['math_on_list'] = function(block) {
     case 'SUM':
       var functionName = Blockly.Dart.provideFunction_(
           'math_sum',
-          [ 'num ' + Blockly.Dart.FUNCTION_NAME_PLACEHOLDER_ +
+          ['num ' + Blockly.Dart.FUNCTION_NAME_PLACEHOLDER_ +
               '(List myList) {',
-            '  num sumVal = 0;',
-            '  myList.forEach((num entry) {sumVal += entry;});',
-            '  return sumVal;',
-            '}']);
+           '  num sumVal = 0;',
+           '  myList.forEach((num entry) {sumVal += entry;});',
+           '  return sumVal;',
+           '}']);
       code = functionName + '(' + list + ')';
       break;
     case 'MIN':
       Blockly.Dart.definitions_['import_dart_math'] =
           'import \'dart:math\' as Math;';
       var functionName = Blockly.Dart.provideFunction_(
           'math_min',
-          [ 'num ' + Blockly.Dart.FUNCTION_NAME_PLACEHOLDER_ +
+          ['num ' + Blockly.Dart.FUNCTION_NAME_PLACEHOLDER_ +
               '(List myList) {',
-            '  if (myList.isEmpty) return null;',
-            '  num minVal = myList[0];',
-            '  myList.forEach((num entry) ' +
+           '  if (myList.isEmpty) return null;',
+           '  num minVal = myList[0];',
+           '  myList.forEach((num entry) ' +
               '{minVal = Math.min(minVal, entry);});',
-            '  return minVal;',
-            '}']);
+           '  return minVal;',
+           '}']);
       code = functionName + '(' + list + ')';
       break;
     case 'MAX':
       Blockly.Dart.definitions_['import_dart_math'] =
           'import \'dart:math\' as Math;';
       var functionName = Blockly.Dart.provideFunction_(
           'math_max',
-          [ 'num ' + Blockly.Dart.FUNCTION_NAME_PLACEHOLDER_ +
+          ['num ' + Blockly.Dart.FUNCTION_NAME_PLACEHOLDER_ +
               '(List myList) {',
-            '  if (myList.isEmpty) return null;',
-            '  num maxVal = myList[0];',
-            '  myList.forEach((num entry) ' +
-                  '{maxVal = Math.max(maxVal, entry);});',
```

Comment:
I'd like to keep the indent on this line since it highlights that this is a wrap of JS, not of Dart.